llama_stack/apis/inference/inference.py

```diff
@@ -202,6 +202,15 @@ class BatchChatCompletionResponse(BaseModel):
     batch: List[ChatCompletionResponse]
 
 
+@json_schema_type
+class EmbeddingRequest(BaseModel):
```

Comment:
I feel like we don't need this intermediate class at all. The reason we needed these classes for ChatCompletionRequest, etc. was because a bunch of functions needed to hold on to a bag of params. It is not the case for embeddings.